addons/debugger/addon.json

```diff
@@ -85,7 +85,7 @@
   "versionAdded": "1.16.0",
   "latestUpdate": {
     "version": "1.33.0",
-    "temporaryNotice": "There is now an option to disable the filling and animating of graphs.",
+    "temporaryNotice": "There's now a low detail mode for graphs to increase performance. Turning on \"high detail graphs\" fills and animates graphs.",
```

Comment:
```suggestion
    "temporaryNotice": "There is now a low-detail mode for graphs to increase performance. Turning on \"high detail graphs\" fills and animates graphs.",
```

Reply:
Keep in mind that this will need to be revised again if Samq64's advice is accepted.

Reply:
Needs to be changed